test/MediaBox.spec.js

```diff
@@ -26,7 +26,7 @@ describe('<MediaBox />', () => {
   });
 
   test('renders', () => {
-    expect(shallow(wrapper)).toMatchSnapshot();
+    expect(mount(wrapper)).toMatchSnapshot();
```

Comment:
sorry to be a pain, does this work with shallow? if so, let's leave shallow

Reply:
It's fine :smile: